sshnp_register_tool/README.md

```diff
@@ -0,0 +1,189 @@
+# sshnp_register_tool
+
+This tool can onboard atSigns easily through the command-line and create the systemd.service (sshnpd.service) file for your device.
+
+# Usage
+
+There are two ways you can use this tool:
+
+1. Dart
+2. Binary
+
+## 1. Dart
+
+Fork the repository (or clone it) and run the tool from the repository itself via `dart run`.
+
+```sh
+dart run bin/sshnp_register_tool.dart -e <email@email.com>
+```
+
+## 2. Binary
```

Comment:
Can we create binaries (using a GitHub Action) and make them available here as releases?